miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/MetadataApi.kt

```diff
@@ -0,0 +1,38 @@
+package com.rakuten.tech.mobile.miniapp.api
+
+import androidx.annotation.Keep
+import com.google.gson.annotations.SerializedName
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+internal interface MetadataApi {
+    @GET("host/{hostId}/miniapp/{miniappId}/version/{versionId}/{testPath}/metadata")
+    fun fetchMetadata(
+        @Path("hostId") hostId: String,
+        @Path("miniappId") miniAppId: String,
+        @Path("versionId") versionId: String,
+        @Path("testPath") testPath: String = ""
+    ): Call<MetadataEntity>
+}
+
+@Keep
+internal data class MetadataEntity(
+    @SerializedName("metadata") val metadata: MetadataResponse?
```

Comment:
TODO: rename to `bundleManifest`.

Reply:
I think you can go ahead and change this to `bundleManifest` so we can get the PR merged. We can make any minor changes later if needed.

Reply:
Yeah, I can address it in a separate PR.

Reply:
I think you should change it in this PR to `bundleManifest` because that is supposed to be the correct format. It will potentially avoid the need for another PR.

Sorry, I meant that if there are any _unexpected_ changes/issues later, then we can make a separate PR.